src/core/graph.test.js

```diff
@@ -1663,4 +1664,87 @@ describe("core/graph", () => {
       expect(edgeToParts(edge)).toEqual(expected);
     });
   });
+
+  describe("compareGraphs", () => {
+    describe("equivalent graphs", () => {
+      const graph1 = new Graph().addNode(src).addNode(dst).addEdge(simpleEdge);
+      const graph2 = new Graph().addNode(src).addNode(dst).addEdge(simpleEdge);
+
+      it("returns empty arrays", () => {
```

Comment:
recommend renaming test case to "equivalent graphs are equal with no differences" or such

Reply:
done